packages/getting-started/README.md

```diff
@@ -1,15 +1,40 @@
-# Theia - Getting Started Extension
+<div align='center'>
+
+<br />
+
+<img src='../../logo/theia.svg' width='100px' />
+
+<h2>THEIA - GETTING-STARTED EXTENSION</h2>
+
+<hr />
+
+</div>
 
 
 ## Description
 
-The extension contributes a `getting-started` widget which is displayed when opening the application
-without a workspace present. The `getting-started` widget provides useful commands and functionality for quickly getting up to speed with the application:
+The `@theia/getting-started` extension contributes a default `getting-started` widget which is displayed whenever
+opening the application without any workspace present or if the widget is explicitly opened (through the menu or command).
+
+The `getting-started` widget provides useful commands and functionality for quickly getting up to speed with the application. For example:
 - `open commands`: commands which are used to open files, folders, and workspaces quickly.
 - `recent workspaces`: recently used workspaces are listed for easy and quick access.
 - `settings commands`: commands which are used to open the preferences and keyboard shortcuts widgets.
 - `help`: useful links pointing to documentation and/or guides.
 
+
+## Additional Information
+
+- [API documentation for `@theia/getting-started`](https://eclipse-theia.github.io/theia/docs/next/modules/getting_started.html)
+- [Theia - GitHub](https://github.com/eclipse-theia/theia)
```

Comment:
@lmcbout I added the following links to the `Additional Information` section for the `readme`.
Did you have something else in mind?